schemas/data.schema.json

```diff
@@ -195,6 +234,43 @@
     "WebFeaturesData": {
       "additionalProperties": false,
       "properties": {
+        "browsers": {
+          "additionalProperties": false,
+          "description": "Browsers and browser release data",
+          "properties": {
+            "chrome": {
+              "$ref": "#/definitions/BrowserData"
+            },
+            "chrome_android": {
+              "$ref": "#/definitions/BrowserData"
+            },
+            "edge": {
+              "$ref": "#/definitions/BrowserData"
+            },
+            "firefox": {
+              "$ref": "#/definitions/BrowserData"
+            },
+            "firefox_android": {
+              "$ref": "#/definitions/BrowserData"
+            },
+            "safari": {
+              "$ref": "#/definitions/BrowserData"
+            },
+            "safari_ios": {
+              "$ref": "#/definitions/BrowserData"
+            }
+          },
+          "required": [
```

Comment:
This is a nice touch, publishing the core browser set in the schema as well, indirectly.